### DISCUSSION TITLE
No conversion at all of acsm file with deacsm and dedrm

### DISCUSSION THREAD

**Discussion by final-option** [noDRM/DeDRM_tools#624] [General]
Hey.

I just installed both of the plugins. 
I created an anonymous ade key and imported that key to deDRM.
When I try to import the book it just adds an acsm file. No epub conversion at all takes place.

Here is the debug log from my console:

calibre 7.15  embedded-python: True
Windows-10-10.0.19045-SP0 Windows ('64bit', 'WindowsPE')
('Windows', '10', '10.0.19045')
Python 3.11.5
Windows: ('10', '10.0.19045', 'SP0', 'Multiprocessor Free')
Interface language: de
EXE path: C:\Program Files\Calibre2\calibre-debug.exe
Successfully initialized third party plugins: DeACSM (0, 0, 16) && DeDRM (10, 0, 9)
QPA platform: windows
devicePixelRatio: 1.25
logicalDpi: 96.0 x 96.0
physicalDpi: 113.41395348837209 x 113.12164948453608
ApplicationPaletteChange event ignored
Using calibre Qt style: True
[0.00] Starting up...
[0.01] Showing splash screen...
[0.17] splash screen shown
[0.17] Initializing db...
[0.20] db initialized
[0.20] Constructing main UI...
[0.62] GUI main window shown
[0.67] main UI initialized...
[0.67] Hiding splash screen
[1.26] splash screen hidden
[1.26] Started up in 1.26 seconds with 1109 books
stdout+stderr from file dialog helper: [b'', b'']
piped data from file dialog helper: [b'\x93BV\x15>\xd8\xdd\xa2a\x9b\xbeY\xe7QV\x8e\x91\xb9B\x07\\\x03\xd5\xca\xc0\xef\xb6)\xe2]\x02o', b'X:\\DiedreiSonnen_9783641173074_3061436.acsm']
Added DiedreiSonnen 9783641173074 3061436 to db in: 0.1
Added 1 books in 2.3 seconds

Any ideas anyone?

Thanks

**Comment by ElleKayEm** [comment]
This isn't a DeDRM issue (at least not yet.)  For help with downloading the epub or pdf via DeACSM, see this thread at MobileRead: https://www.mobileread.com/forums/showthread.php?t=341975

**Reply by final-option** [reply]
Hey, thanks for your quick answer.
I managed to get a proper log file (did not know I have to close calibre).
It seems like deACSM has a problem as you pointed out in your answer. 
Unfortunately I could not find a solution in the discussion that you linked to.
Here again the log - should I post this at the deACSM discussion page?

ApplicationPaletteChange event ignored
Using calibre Qt style: True
calibre Fehlerdiagnoseprotokoll
calibre 7.17  embedded-python: True
Windows-10-10.0.19045-SP0 Windows ('64bit', 'WindowsPE')
('Windows', '10', '10.0.19045')
Python 3.11.5
Windows: ('10', '10.0.19045', 'SP0', 'Multiprocessor Free')
Interface language: de
EXE path: C:\Program Files\Calibre2\calibre-debug.exe
Successfully initialized third party plugins: DeACSM (0, 0, 16) && DeDRM (10, 0, 9)
calibre 7.17  embedded-python: True
Windows-10-10.0.19045-SP0 Windows ('64bit', 'WindowsPE')
('Windows', '10', '10.0.19045')
Python 3.11.5
Windows: ('10', '10.0.19045', 'SP0', 'Multiprocessor Free')
Interface language: de
EXE path: C:\Program Files\Calibre2\calibre-debug.exe
Successfully initialized third party plugins: DeACSM (0, 0, 16) && DeDRM (10, 0, 9)
QPA platform: windows
devicePixelRatio: 1.25
logicalDpi: 96.0 x 96.0
physicalDpi: 113.41395348837209 x 113.12164948453608
[0.00] Starting up...
[0.00] Showing splash screen...
[1.20] splash screen shown
[1.20] Initializing db...
[1.25] db initialized
[1.25] Constructing main UI...
[1.67] GUI main window shown
[1.72] main UI initialized...
[1.72] Hiding splash screen
[1.72] splash screen hidden
[1.72] Started up in 1.72 seconds with 1109 books
stdout+stderr from file dialog helper: [b'', b'']
piped data from file dialog helper: [b'\x96\xf4\xaf\xe8\xcc:\xef"&\xd9\x83E\xb4\xd9\x96\xe5\xd8<\xf2\xe3i>\xb8 \xc3\xc4\x85\xdaC\'\xc9\x1b', b'X:\\DiedreiSonnen_9783641173074_3061436.acsm']
**DeACSM v0.0.16: Trying to parse file DiedreiSonnen_9783641173074_3061436.acsm
DeACSM v0.0.16: Try to fulfill ...
DeACSM v0.0.16: Hey, that didn't work: 
Looks like there's been an error during Fulfillment: <error xmlns="http://ns.adobe.com/adept" data="E_ADEPT_REQUEST_EXPIRED http://acsepub.contentreserve.com/fulfillment/Fulfill 2024-08-31T15:06:54-04:00%20(1725131214000)%20is%20before%202024-09-06T17:21:41-04:00%20(1725657701379)"/>**

**Reply by final-option** [reply]
I have opened a discussion over at deACSM ...

**Reply by ElleKayEm** [reply]
Yeah, I have no experience with DeACSM.  I use Adobe Digital Editions myself.
